libbpf-tools/drsnoop.h

```diff
@@ -0,0 +1,15 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+#ifndef __DRSNOOP_H
+#define __DRSNOOP_H
+
+#define TASK_COMM_LEN 16
+
+struct event {
+	char task[TASK_COMM_LEN];
+	__u64 delta_ns;
+	__u64 nr_reclaimed;
+	pid_t pid;
```

Comment:
nit: this leaves 4 bytes of padding between pid and nr_free_pages, maybe move pid to the very end, so when new fields are added, they can fill in the padding?

Reply:
Yes, it's my fault. Only change `nr_reclaimed` from `__u32` to `__64`,  forgot to check padding things.